vmm/src/vmm_config/vsock.rs

```diff
@@ -13,6 +13,8 @@ pub struct VsockDeviceConfig {
     pub id: String,
```

Comment:
Not clear what the purpose of this is.

Reply:
It's just a device identifier, since currently there's support for multiple devices, even if the Linux driver doesn't.